pages/glossary/non-repeated-content.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Non-repeated content after repeated content
```

Comment:
I don't think this should be a definition. This isn't a concept that needed a clearer definition, as much as it's a way to avoid repeating something in the rule. I don't think that's how definitions should be used.

Reply:
I'm sticking with having it as a definition. First, it does clarify things a bit (for example by explaining that "after" means "in tree order in the flat tree" rather than, say "in visual order"). Second, I disagree that shortening long periphrases is a bad usage for definition (we have many definitions that are a one line sentence and could just be inlined).

In that case, we could replace all occurrences of "non-repeated content after repeated content" by "perceivable content not in a block of repeated content after (in tree order in the flat tree) at least on block of repeated content" but I think that it would make the rules completely unreadable.